aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -1191,7 +1191,10 @@ Tensor try_get_weight_buf(
   // Try to get parameter storage
   auto & any_param = parameters.at(0);
   auto param_storage = any_param.storage();
-  auto weight_buf = at::empty({0}, any_param.options()).set_(param_storage);
+  auto param_offset = any_param.storage_offset();
+  std::initializer_list<int64_t> buffer_size = {
```

Comment:
why is this 2-dimensional?  Wasn't it previously 1-dimensional?

Reply:
you're right, it should be 1-dim